scripts/remove_dups.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/python3
+#
+# remove_dups.py
+
+import json, os
+
+links = []
+parent = os.path.split(os.path.abspath(os.getcwd()))[0]
+data = open(os.path.join(parent, 'data/data.json'), 'r')
+config = json.load(data)
+data.close()
+
+for paper in config:
+    links.append(paper['Link'])
```

Comment:
Better to use a dict here. I will make the change and merge! Thanks for this template. :+1: